apps/form-app-e2e/.eslintrc.json

```diff
@@ -1,10 +1,29 @@
 {
-  "extends": ["plugin:cypress/recommended", "../../.eslintrc.json"],
-  "ignorePatterns": ["!**/*"],
+  "rules": {
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+    "cypress/no-unnecessary-waiting": "off",
+    "@typescript-eslint/no-non-null-assertion": "off",
+    "@typescript-eslint/no-explicit-any": "off"
+  },
+  "extends": [
+    "plugin:cypress/recommended",
+    "../../.eslintrc"
+  ],
   "overrides": [
     {
-      "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
-      "rules": {}
+      "files": [
+        "src/plugins/index.js",
+        "src/support/multiple-cucumber-html-reporter.js",
+        "src/support/index.ts"
+      ],
+      "rules": {
+        "@typescript-eslint/no-var-requires": "off",
+        "@typescript-eslint/camelcase": "off",
+        "no-undef": "off"
+      }
     }
+  ],
+  "ignorePatterns": [
+    "!**/*"
   ]
-}
+}
```

Comment:
remove space at the end of the file

Reply:
This file doesn't have empty space or empty line in the end.